stdlib/Statistics/test/runtests.jl

```diff
@@ -370,6 +370,12 @@ Y = [6.0  2.0;
         @test C ≈ Cxy
         @inferred cov(X, Y, dims=vd, corrected=cr)
     end
+
+    # case for large number, floating point accurcy issue
```

Comment:
Would be great to make this comment the title of a `testset`. It helps when the test fails.

Reply:
ah, okay